app/components/chat/AssistantMessage.tsx

```diff
@@ -20,11 +27,18 @@ export const AssistantMessage = memo(({ content, annotations }: AssistantMessage
 
   return (
     <div className="overflow-hidden w-full">
-      {usage && (
-        <div className="text-sm text-bolt-elements-textSecondary mb-2">
-          Tokens: {usage.totalTokens} (prompt: {usage.promptTokens}, completion: {usage.completionTokens})
+      <>
+        <div className=" flex gap-2 items-center text-sm text-bolt-elements-textSecondary mb-2">
+          {progressAnnotation.length > 0 && (
```

Comment:
Is this the progress of the context being compiled?

Reply:
yes, the idea is to show this realtime in the UI but I am not able to present it realtime, 
its arriving to the ui when the streaming started. still figuring out how can I show it realtime.